src/router/router.ts

```diff
@@ -176,6 +186,21 @@ export function createRouter(baseGuiPath: string = '/', policyDefinitions: Polic
             },
           ],
         },
+        {
+          path: 'policies',
+          name: 'policies',
```

Comment:
**Suggestion**: Adds the title needed to have this show up in the breadcrumbs correctly (related to the suggestion with the `parent` field).

```suggestion
          name: 'policies',
          meta: {
            title: 'Policies',
          },
```